Toolchain/BuildGNU.sh

```diff
@@ -246,6 +246,7 @@ pushd "$DIR/Build/$ARCH"
                                                  --enable-static \
                                                  --disable-shared \
                                                  --disable-nls \
+                                                 --with-system-zlib \
```

Comment:
commit message: What actually happened is that Apple-Clang has now picked up https://github.com/llvm/llvm-project/pull/74676, which required fixes to several popular libraries. One of them being zlib, which got fixed in https://github.com/madler/zlib/pull/895. binutils's and gcc's hermetic zlib apparently don't have that commit yet, but system zlib does.

If binutils and gcc pick up that fix, this here is no longer necessary (but shouldn't hurt either).

Reply:
Ah, Thanks for finding out the real cause,
Will amend the commit message soon:tm:

Reply:
commit message grammar is a bit off. s/but the/the/ is one way to fix.

Reply:
Currently in a  different build, will amend shortly

Reply:
Done